comm/src/main/scala/coop/rchain/comm/package.scala

```diff
@@ -13,6 +18,20 @@ package object comm {
 
   val CommMetricsSource: Metrics.Source = Metrics.Source(Metrics.BaseSource, "comm")
 
+  private def validateInetAddress[F[_]: Sync](host: String, p: InetAddress => Boolean): F[Boolean] =
+    Sync[F].delay(InetAddress.getByName(host)).attempt.map(_.fold(kp(false), p))
+
+  def isValidPublicInetAddress[F[_]: Sync](host: String): F[Boolean] =
```

Comment:
Package file is not the right place to define helper functions. For now it can be in HandleMessages, but we need better way of organizing files them we have now.